db/migrate/20161121200712_add_category_id_to_tags.rb

```diff
@@ -0,0 +1,5 @@
+class AddCategoryIdToTags < ActiveRecord::Migration
+  def change
+    add_column :tags, :category_id, :integer
```

Comment:
This could use an index, if only to be consistent with the other foreign key columns that are also indexed. And maybe declaring it as not allowing null would make sense, but a lot of other columns in this project's schema also allow null, so it really isn't an issue.